docs/usage/quickstart.rst

```diff
@@ -3,112 +3,112 @@
+have at least one disk available to provision storage volumes.
 
 .. todo:: Give some sizing examples
 
 .. _MetalK8s: https://github.com/scality/metal-k8s/
+
 .. _CentOS: https://www.centos.org
+
 .. _Kubernetes: https://kubernetes.io
 
 Defining an Inventory
 ---------------------
-To tell the Ansible_-based deployment system on which machines MetalK8s should
-be installed, a so-called *inventory* needs to be provided. This inventory
-contains a file listing all the hosts comprising the cluster, as well as some
-configuration.
+To tell the Ansible_-based deployment system which machines MetalK8s should be
+installed on, you must provide an *inventory*. This inventory contains a file
+that lists all hosts in the cluster, as well as some configuration.
 
 .. _Ansible: https://www.ansible.com
 
-First, create a directory, e.g. :file:`inventory/quickstart-cluster`, in which
-the inventory will be stored. For our setup, we need to create two files. One
-listing all the hosts, aptly called :file:`hosts`:
+To create an inventory:
+
+1. Create a directory (for example,
+:file:`inventory/quickstart-cluster`) in which the inventory will be stored.
 
-.. code-block:: ini
+2. Create two files:
 
-    node-01 ansible_host=10.0.0.1 ansible_user=centos
-    node-02 ansible_host=10.0.0.2 ansible_user=centos
-    node-03 ansible_host=10.0.0.3 ansible_user=centos
+  - The first, called :file:`hosts`:, lists all hosts::
 
-    [kube-master]
-    node-01
-    node-02
-    node-03
+         node-01 ansible_host=10.0.0.1 ansible_user=centos
+         node-02 ansible_host=10.0.0.2 ansible_user=centos
+         node-03 ansible_host=10.0.0.3 ansible_user=centos
 
-    [etcd]
-    node-01
-    node-02
-    node-03
+         [kube-master]
+         node-01
+         node-02
+         node-03
 
-    [kube-node]
-    node-01
-    node-02
-    node-03
+         [etcd]
+         node-01
+         node-02
+         node-03
 
-    [k8s-cluster:children]
-    kube-node
-    kube-master
+         [kube-node]
+         node-01
+         node-02
+         node-03
 
-Make sure to change IP-addresses, usernames etc. according to your
-infrastructure.
+         [k8s-cluster:children]
+         kube-node
+         kube-master
 
-In a second file, called :file:`kube-node.yml` in a :file:`group_vars`
-subdirectory of our inventory, we declare how to setup storage (in the
-default configuration) on hosts in the *kube-node* group, i.e. hosts on which
-Pods will be scheduled:
+    Make sure to change IP addresses, user names, etc. to conform to your
+    infrastructure.
 
-.. code-block:: yaml
+  - A second file, called :file:`kube-node.yml`, in a :file:`group_vars`
+   subdirectory of the inventory, declares how to set up storage (in the
+   default configuration) on hosts in the *kube-node* group, that is,
+   hosts on which Pods will be scheduled::
 
-    metal_k8s_lvm:
-      vgs:
-        kubevg:
-          drives: ['/dev/vdb']
+         metal_k8s_lvm:
```

Comment:
YAML formatting is no longer specified.

Reply:
Are you saying my edits broke yml formatting, or that yml formatting is no longer required?

Either way, how do you want this passage changed?

As it stands, it's five spaces (minimum to trigger preformat in the Atom parser) plus two additional spaces for each following line. This renders the same as what's evident in the original, only it's nested under an unordered list item.

Reply:
@Stefano explained. Thanks!